apigw-lambda-healthscribe-sam/README.md

```diff
@@ -0,0 +1,173 @@
+# Amazon API Gateway to AWS Lambda to AWS HealthScribe using SAM
+
+This sample project deploys an Amazon API Gateway REST API with an AWS Lambda integration. The Lambda function is written in Python, calls the AWS HealthScribe API to automatically generate clinical notes from patient-clinician conversations.
+
+Learn more about this pattern at Serverless Land Patterns: serverlessland.com/patterns/apigw-lambda-healthscribe-sam
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+- [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Prerequisite
+This code asset has dependency on boto3 v1.34.26 or above. This is not the current default version of boto3 on AWS Lambda. Hence we will use Lambda layer. Please refer [API Documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/transcribe/client/start_medical_scribe_job.html).  
```

Comment:
```suggestion
This code asset has a dependency on boto3 v1.34.26 or above. This is not the current default version of boto3 on AWS Lambda. Hence we will use a Lambda layer. Please refer [API Documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/transcribe/client/start_medical_scribe_job.html).  
```

Reply:
Was the link intended to point to the Lambda layer documentation?

Reply:
I changed the link location to avoid confusion.